fizzbuzz.js

```diff
@@ -1,21 +1,27 @@
+const statusCodes = require('./statusCodes.js');
+
 const fizzbuzz = (number) => {
+  if (typeof number === 'undefined')
+    return (statusCodes.NullValue);
```

Comment:
don't need parenthesis around this; you can just do `return statusCodes.nullValue`